docs/usage/java.md

```diff
@@ -38,31 +38,17 @@ Renovate will search repositories for all `pom.xml` files and processes them ind
 
 ### Custom registry support, and authentication
 
-Here is an example configuration to work with custom Artifactory servers using authentication:
+In the above config, the custom registry URLs are defined using a package rule, and the username/passwords are set using a host rule each. If you don't want to store your artifactory credentials in plaintext, you can pass them as an environment variable using a javascript config file like `config.js`:
 
-```json
-{
-  "hostRules": [
+```js
+module.exports = {
+  hostRules: [
     {
-      "hostType": "maven",
-      "baseUrl": "https://artifactoryurl1/",
-      "username": "artifactoryusername",
-      "password": "artifactorypassword"
+      hostType: 'maven',
+      baseUrl: 'https://artifactory.yourcompany.com/',
+      username: process.env.ARTIFACTORY_USR,
+      password: process.env.ARTIFACTORY_PSW,
```

Comment:
```suggestion
      password: process.env.ARTIFACTORY_TOKEN,
```

Artifactory seems to supports token based authentication as well (https://www.jfrog.com/confluence/display/JFROG/Access+Tokens). I'm not sure if this is easy/hard to do, and if this is well supported for all Artifactory offerings (JFrog Cloud, JFrog Self-Hosted).

So somebody who actually uses Artifactory should decide whether we want to use tokens or passwords in our example config.